src/Ilios/CoreBundle/Entity/Course.php

```diff
@@ -66,6 +68,7 @@ class Course implements CourseInterface
      *      max = 200
      * )
      *
+     * @Gedmo\Versioned
```

Comment:
i don't see this annotation being used elsewhere? leftover from some abandoned attempt, or legit?